src/lib/openapi/util/create-response-schema.ts

```diff
@@ -1,20 +1,41 @@
 import { OpenAPIV3 } from 'openapi-types';
 
-export const createResponseSchema = (
-    schemaName: string,
+export const createResponseSchemas = (
+    description: string,
+    content: { [media: string]: OpenAPIV3.MediaTypeObject },
 ): OpenAPIV3.ResponseObject => {
     return {
-        description: schemaName,
-        content: {
-            'application/json': {
-                schema: {
-                    $ref: `#/components/schemas/${schemaName}`,
-                },
-            },
+        description,
+        content: content,
+    };
+};
+
+export const schemaNamed = (schemaName: string): OpenAPIV3.MediaTypeObject => {
+    return {
+        schema: {
+            $ref: `#/components/schemas/${schemaName}`,
+        },
+    };
+};
+
+export const schemaTyped = (
+    type: OpenAPIV3.NonArraySchemaObjectType,
+): OpenAPIV3.MediaTypeObject => {
+    return {
+        schema: {
+            type,
         },
     };
 };
```

Comment:
These 2 are helper methods to keep the code short as here:
https://github.com/Unleash/unleash/pull/3247/files#diff-01bbb8bc0ca5a5d90fce151fb09d8b1f16d4b0821b89aca998567341aeaa88f8R25-R28